docs/flutterinfinitelisttutorial.md

```diff
@@ -130,29 +130,27 @@ Let’s create `post_bloc.dart` and create our empty `PostBloc`.
 
 ?> **Note:** Just from the class declaration we can tell that our PostBloc will be taking PostEvents as input and outputting PostStates.
 
-Next, we need to implement `mapEventToState` which will be fired every time a `PostEvent` is added.
+Next, we need to register handlers for as many `PostEvent` we have in our bloc. In this case we will register a handler for `PostFetched`. This handler will call `_fetchPosts` that will get the data from API.
 
-[post_bloc.dart](_snippets/flutter_infinite_list_tutorial/post_bloc_map_event_to_state.dart.md ':include')
+[post_bloc.dart](_snippets/flutter_infinite_list_tutorial/post_bloc_on_post_fetched.dart.md ':include')
```

Comment:
can we update this link in the other tutorials as well (docs/fr/flutterinfinitelisttutorial.md, etc)? Otherwise merging these changes will break the translated versions of the tutorial.